api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,273 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](*) and [devices](*) (and more?) who share access to resources in Office365 or in your app. All operations in Microsoft Graph on groups require administrator consent. Using Microsoft Graph, you can perform the following operations and more:

```

Comment:
Agreed. I'll try adding in a "Common scenarios" table that is more helpful instead of this flat list.